examples/supported-python-versions-example/control-service-setup/README.md

```diff
@@ -0,0 +1,258 @@
+Supported Python Versions Example PART 1 - Control Service Setup
+
+Overview
+--------
+
+This is the first part of the "Supported Python Versions" example. In this part, we will deploy a Control Service
+instance in a local kubernetes cluster using vdk server, and will update the service to support python3.9 in two
+different vdk configurations, `3.9-pre-release` and `3.9-release`. These would be based on the quickstart-vdk images
```

Comment:
It would be nice to add one short sentence explaining the problem we are solving here.

Reply:
Done, thanks